watson-developer-cloud/discovery-components
# fix: simpleDoc passage highlighting

#### What do these changes do/fix?

This fixes the passage highlighting in `SimpleDocument` when the highlight location is more than 64KB worth of characters. 

#### How do you test/verify these changes?

Run storybook and go to DocumentPreview.

Select the passage highlighting story and select the multiline knob

See that `32.7 Buyer’s and Customer's Regulatory Authorities shall have the benefit of any rights of audit and access to information and documentation provided for in this Agreement to the extent that they relate to the exercise of the Regulatory Authorities' legal rights and/or responsibilities.` is highlighted.
